github/Repository.py

```diff
@@ -1207,10 +1207,10 @@ def create_git_tag_and_release(
         self,
         tag: str,
         tag_message: str,
-        release_name: str,
-        release_message: str,
         object: str,
         type: str,
+        release_name: Opt[str] = NotSet,
```

Comment:
This is a breaking change, but I cannot see how to avoid that...

Reply:
Please adjust the order of the `:param`s in the docstring.

Reply:
I am afraid we have to revert this breaking change. Please replace
```
        release_name: str,
        release_message: str,
```
with
```
        release_name: Opt[str],
        release_message: Opt[str],
```
instead.

Calling code will have to explicitly provide `release_name=None, release_message=None`.